### PR TITLE
Misuv 3968 add money in your account link

### PR DESCRIPTION
**Pull request by tharunprasath**

### CODE REVIEW

File: app/views/WhatYouOwe.scala.html [hunk 1]
```diff
@@ -375,6 +393,22 @@
             @messages("whatYouOwe.payments-made-bullet-1.3")</li>
         <li> @messages("whatYouOwe.payments-made-bullet-2")</li>
     </ul>
+
+    @if(creditAndRefundEnabled && whatYouOweChargesList.balanceDetails.availableCredit.isDefined && whatYouOweChargesList.balanceDetails.availableCredit.get > 0) {
+        <div class="govuk-body" id="money-in-your-account">
+        @if(isAgent){
```
Comment (joesheridan) [inline app/views/WhatYouOwe.scala.html:399]: minor comment: should be space before left brace.

**Review by joesheridan — CHANGES_REQUESTED** [review]

**Comment by joesheridan** [comment]
Also we need unit tests for the whatyouowe controller and view

File: app/views/WhatYouOwe.scala.html (hunk 1 above)
Comment (tharunprasath) [reply app/views/WhatYouOwe.scala.html]: Done.

File: app/views/WhatYouOwe.scala.html [hunk 2]
```diff
@@ -375,6 +393,22 @@
             @messages("whatYouOwe.payments-made-bullet-1.3")</li>
         <li> @messages("whatYouOwe.payments-made-bullet-2")</li>
     </ul>
+
+    @if(creditAndRefundEnabled && whatYouOweChargesList.balanceDetails.availableCredit.isDefined && whatYouOweChargesList.balanceDetails.availableCredit.get > 0) {
```
Comment (joesheridan) [inline app/views/WhatYouOwe.scala.html:397]: might be nicer like this:
whatYouOweChargesList.balanceDetails.availableCredit.map( value => {
    if(value > 0) {
        ...something
    }
}

**Review by joesheridan — APPROVED** [review]

**Pull request merged**